sourcetools/com.ibm.uma/com/ibm/j9/uma/platform/PlatformWindows.java

```diff
@@ -38,6 +38,7 @@
 public class PlatformWindows extends PlatformImplementation {
 	
 	static final String DEFAULT_RC_PRODUCT_NAME="IBM SDK, Java(tm) 2 Technology Edition";
+	static final String VERSION_MAJOR = System.getenv("VERSION_MAJOR");
```

Comment:
Is this already guaranteed to be set in the environment or is there another change to the build files to add this env var?

Reply:
Note this will have a value of 10 not 10.0.1.0. To get the actual version requires changes to the extensions code in order to make the appropriate version info available. J9VERSION_STRING embeds the version (as 10.0.1) so looking at how this works would be a start.

Reply:
it looks like the default if its not set is "9" if its not set https://github.com/eclipse/openj9/blob/master/runtime/makelib/mkconstants.mk.ftl#L27.  should that be 11 now?

Reply:
It should always be set by the extensions, not sure if it matters what the default is.

Reply:
Since we expect it to be set by the OpenJDK extensions, probably an error should occur if the value is not set.